test/quantization/core/test_workflow_module.py

```diff
@@ -1118,6 +1123,52 @@ def test_fused_mod_reduce_range(self):
         self.assertEqual(obs.quant_min, 0)
         self.assertEqual(obs.quant_max, 127)
 
+    def test_embedding_bag_qat_config(self):
+        class Model(nn.Module):
+            def __init__(self):
+                super(Model, self).__init__()
+                self.emb1 = torch.nn.EmbeddingBag(num_embeddings=10, embedding_dim=12,
+                                                  include_last_offset=True, scale_grad_by_freq=False, mode='sum')
+                self.emb2 = torch.nn.EmbeddingBag(num_embeddings=10, embedding_dim=12,
+                                                  include_last_offset=True, scale_grad_by_freq=False, mode='sum')
+                # Note for EmbeddingBag, indices should never be quantized, only outputs
+                self.dequant = torch.quantization.DeQuantStub()
+
+            def forward(self, indices):
+                x = torch.cat((self.emb1(indices), self.emb2(indices)))
+                return self.dequant(x)
+
+        model = Model()
+        indices = torch.randint(0, 10, (5, 12))
+        qat_mappings = get_default_qat_module_mappings()
+        float_qparams_fakequant = FakeQuantize.with_args(observer=PerChannelMinMaxObserver,
+                                                         qscheme=torch.per_channel_affine_float_qparams,
+                                                         ch_axis=0,
+                                                         memoryless=True)
+        float_qparams_qconfig = QConfig(activation=NoopObserver,
+                                        weight=float_qparams_fakequant)
+        model.qconfig = float_qparams_qconfig
+
+        quant_model = torch.quantization.prepare_qat(model, mapping=qat_mappings)
```

Comment:
I think we do not need to provide qat_mappings here if there is no additional mapping we add to default ones? it should be picked up by prepare_qat directly: https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/quantize.py#L415